samples/cellular/modem_shell/src/link/link_shell.c

```diff
@@ -357,6 +358,40 @@ static const char link_propripsm_usage_str[] =
 	"  -e, --enable,       Enable proprietary PSM\n"
 	"  -h, --help,         Shows this help information";
 
+static const char link_uiccpowersave_usage_str[] =
+	"Usage: link uiccpowersave [options] | --read\n"
+	"Options:\n"
+	"  -r, --read,         Read and print the current UICC power saving mode.\n"
+	"      --default_mode, Default functionality.\n"
+	"                      The UICC is deactivated during eDRX only when:\n"
+	"                      - It is allowed by the EF-AD (Elementary File - Administrative\n"
+	"                        Data)\n"
+	"                      - Application PIN is disabled\n"
+	"                      - Sleep mode time is longer than the minimum time limit (1 minute\n"
+	"                        in ultra low power mode and 5 minutes in other power modes)\n"
+	"                      Note: AT&T has a shorter minimum time of 20.48 seconds.\n"
+	"                      Proactive polling interval can be modified during eDRX sleep only\n"
+	"                      if allowed in EF-AD.\n"
+	"                      Supported with modem firmware versions >= 2.0.2.\n"
```

Comment:
Instead of repeating the version requirement for each option, we should rather have one note for the whole command. If MFW doesn't support the feature, also `--read` would fail.